### ISSUE TITLE
Unknown device

### ISSUE THREAD

**Issue by felippenardi** [lprhodes/homebridge-broadlink-rm#539]
I’m getting this:

```
[Info] We've discovered an unknown Broadlink device. This likely won't cause any issues.

Please raise an issue in the GitHub repository (https://github.com/lprhodes/homebridge-broadlink-rm/issues) with details of the type of device and its device type code: "6070". The device is connected to your network with the IP address "192.168.0.104".
```

Do you guys know what is going on?

**Comment by lprhodes** [comment]
Hey, it's most likely just a Broadlink plug or something else on your network. If you can find out what device is connected to IP 192.168.0.104 then I can add it to the exclusion list to hide the error

**Comment by felippenardi** [comment]
It is actually my BroadLink rm! It is not recognizing it. Any time I try scanning it says “device not found”, but it has this warning in the beginning.

**Comment by lprhodes** [comment]
Are you 100% sure, 6070 is really odd code for an RM Pro. Did you get it recently?

**Comment by felippenardi** [comment]
Yes, I just got it recently and the IP shown in this warning matches with the Broadlink app IP.

This is the link from where I bought it: https://pt.aliexpress.com/item/4000221676120.html?srcSns=sns_Copy&tid=white_backgroup_101&mb=yjkqwjeBZ4dJRVr&businessType=ProductDetail&businessType=ProductDetail&spreadType=socialShare&tt=sns_Copy&rdtUrl=https%3A%2F%2Fwww.aliexpress.com%2Fitem%2F4000221676120.html%3F%26srcSns%3Dsns_Copy%26tid%3Dwhite_backgroup_101%26mb%3DyjkqwjeBZ4dJRVr%26businessType%3DProductDetail%26spreadType%3DsocialShare&aff_request_id=eac11668b2be40bca76b3fd5b6eed4d3-1579348890617-05056-_svzWuT&aff_platform=default&cpt=1579348890617&sk=_svzWuT&aff_trace_key=eac11668b2be40bca76b3fd5b6eed4d3-1579348890617-05056-_svzWuT&platform=AE&terminal_id=270c003acf434b019ed4a0cb82c9c7c4

**Comment by felippenardi** [comment]
@lprhodes Do the device have to be included in the list?

**Comment by felippenardi** [comment]
I guess it should be marked as a bug, not "info requested"

**Comment by felippenardi** [comment]
@lprhodes Can you give an update on this?

**Comment by felippenardi** [comment]
@lprhodes Is there any other test or information I can provide you to help investigate it?
